docs/source/format/Columnar.rst

```diff
@@ -1385,6 +1385,37 @@ have two entries in each RecordBatch. For a RecordBatch of this schema with
     buffer 13: col2    data
 
 
+Compression
+-----------
+
+There are three different options for compression of record batch
+body buffers: Buffers can be uncompressed, buffers can be
+compressed with the``lz4`` compression codec, or buffers can
```

Comment:
How about specifying that this ```` ``lz4`` ```` means LZ4 frame format ( https://github.com/lz4/lz4/blob/dev/doc/lz4_Frame_format.md ) not LZ4 block format ( https://github.com/lz4/lz4/blob/dev/doc/lz4_Block_format.md ) explicitly?

See also: https://github.com/apache/arrow/blob/50219ef69f712ca12fa5c85367a4cae9776d1c99/format/Message.fbs#L46-L49